CMakeLists.txt

```
@@ -107,18 +107,33 @@ else()
     endif()
   endif()
 
-  # No config file for this
   if(WITH_GFLAGS)
-    find_package(gflags REQUIRED)
+    # Config with namespace available since gflags 2.2.2
+    option(GFLAGS_USE_TARGET_NAMESPACE "Use gflags import target with namespace." ON)
+    find_package(gflags CONFIG)
+    if(gflags_FOUND)
+      if(DEFINED ${GFLAGS_TARGET})
```

```suggestion
      if(DEFINED GFLAGS_TARGET)
```

please note what we want to check is the *name* of "GFLAGS_TARGET", not the value of `GFLAGS_TARGET`.

---

probably 
```cmake
if(TARGET ${GFLAGS_TARGET})
```

is better?

---

Aha good catch!

---

fixed